README.md

```diff
@@ -4,6 +4,15 @@ A container to create disk images from [bootc](https://github.com/containers/boo
 especially oriented towards [Fedora/CentOS bootc](https://docs.fedoraproject.org/en-US/bootc/) or
 derivatives.
 
+## 📊 Project
+
+- **Website**: <https://www.osbuild.org>
+- **Bug Tracker**: <https://github.com/osbuild/bootc-image-builder/issues>
+- **Matrix**: #image-builder on [fedoraproject.org](https://matrix.to/#/#image-builder:fedoraproject.org)
+* **Discussions**: https://github.com/orgs/osbuild/discussions
+- **Mailing List**: <image-builder@redhat.com>
```

Comment:
This might be a hot take, but I think we should remove the mailing list from all repositories. No one knows how to subscribe, nor how to access archives. I'm fine living without a mailing list myself.